storage/ceph.py

```diff
@@ -1,14 +1,54 @@
 import pyarrow.parquet as pq
-import s3fs
+import re
+import logging
+import pickle
+import s3fs, datetime, os
```

Comment:
If you'd be interested in nice Python, I suggest implementing (using) PEP8. :wink: 

For example here, [about imports](https://www.python.org/dev/peps/pep-0008/#imports) it says:
- Imports should usually be on separate lines
- Imports should be grouped in the following order: Standard library imports, Related third party imports, Local application/library specific imports